masterlist.yaml

```
@@ -9121,6 +9121,10 @@ plugins:
   - name: 'Crossbow Integration - USSEP.esp'
     tag: [ Text ]
 
+  - name: 'Curse of the Dragonborn - REDUX'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/104477/' ]
```

A plugin's `name` node value needs to be the plugin's filename. You can use that name for the location though:

```suggestion
  - name: 'OPHybrid.esp'
    url:
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/104477/'
        name: 'Curse of the Dragonborn - REDUX'
```